tools/migration/internal/utils.go

```diff
@@ -0,0 +1,25 @@
+package internal
+
+import (
+	"os/user"
+	"strings"
+	"time"
+)
+
+// NowString creates an ISO8601-like timestamp string
+func NowString() string {
+	now := time.Now()
+	return now.Format("2006-01-02_150405")
+}
+
+// expandHomedir replaces an initial tilde in a dirname to the home dir.
+// if there is no initial ~ , it returns dirname.
+func ExpandHomedir(dirname string) string {
+	if strings.LastIndex(dirname, "~") != 0 {
+		return dirname
+	}
+
+	usr, _ := user.Current()
+	home := usr.HomeDir
+	return strings.Replace(dirname, "~", home, 1)
+}
```

Comment:
I had looked at go-homedir, but I just now saw a note in the README about how os/user requires `cgo`, which hurts cross-platform compatibility, so that's an additional good reason.